vars/provisionNodes.groovy

```diff
@@ -328,7 +319,7 @@ EOF'''
                                                    libcmocka python-pathlib     \
                                                    python2-numpy git            \
                                                    python2-clustershell         \
-                                                   golang-bin'''
+                                                   golang-bin ipmctl'''
```

Comment:
I guess all of the above are not in the profile image (yet)?  Will they be?

Reply:
They are in the new "ci" profiles for el7 and opensuse15, but not all are known to be in the existing "test" profile in production.
We have some challenges for el8 support that will need to be worked out before then.